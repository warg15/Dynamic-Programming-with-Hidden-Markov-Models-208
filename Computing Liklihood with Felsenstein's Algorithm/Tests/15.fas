>Great-cormorant
GCAACTGCTGTATGAGTAGAATTCGAGGATTGACGATATTCCTTGAAGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCAGGTTAACTTCCCATCCGCTGAGGGACAATAATGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTTGCGAGCAGACTATCAAGAACTGGA
>Bald-eagle
GTAACTGCTGTATGAGTAGAATTTCAGGATTGAGGGTATTTCTTGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACCCGGCTCAACCCAGGTTACCTTCCCATCCGCTGAGGGATAATAATGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAATTGGA
>Dalmatian-pelican
GTAACTGCTGTATGAGTAGAATTTGAGGATGGAGGGTATTTCTTGAGGAGTCCAGCACCGTCAGAAGCTGTAATAATAGAACGCTGCTCAACCCAGGTTACCTTCCCATCCGCTGCGGGACAATAATGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAACTGGA
>Annas-hummingbird
GTAACTGCTGTATGAGTAGAATCTTAGAATTGAGAGTACTCCTTGAGTGGCCCGCCACCGTCACAAGCTGGAATAAACGAACGGGGCTCAACTCAGGTTACCTTCCCATCCACTGAGGGACAATAATGAGAAAGTCGGAATTGTTGTAACTGGTCCACCAGATCCGGATTGTGTTAGAGAACAGACTATCAAGAACTGGA
>Carmine-bee-eater
GTAACTGCTGTATGAGTAGGATTTAGGGATTGAGGGTATTTCTTGAGGAGCCCAGCACCGTCAGAAGCTGGAATAATAAAACGCTGCGCGATACAGGTGACCTTCCCATCCGCTCAGGGATAATAAGGAGGAAACCGGAATTGTTGTAATTGGTAGTGCAGATCCGGGGTATGTTGGCGAACAGACTACCAAAAACTAGA
>Downy-woodpecker
GTAACTGCTGTATGGGTAAGATTTGAGGATTGAGGGTACTTCCTGAGGAGTTCGGGACTGTCAGGAGCTGGAATAATAGACCGCTGATCAACCCAGGTTACCTTCTCATGCGCCAAGGGACAATAATGAAGAAACCGGAATTATCGTAATTGGTCGGACAGATCCGGCGTACGTTAGCGAATAGACTATCAAGAACTGGA
>Grey-crowned-crane
GTAACTGCTATATGAGTAGAACTTGAGGATTGAGGTTATTTCTTGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCGGATTACCTTCCCATCCGTTGAGGGACAATACTGAGGAAACCAGAATTGTCGTAATTGGTCGAACAGATCCGGCTTACGTTAGCGAACAGACTATCAAGAACTGGA
>Common-cuckoo
GTAACTGCTGAATGAATAGAATTTAAAGATTGAGGGTATTTCTTGAGGAGTCCAGCATCGTCAGAAGCTGGGATAATAGAAAGCTGCTCAAGCTGGACTACCTTCCCATCCGCTGAGGGACAATAATGAGGAAACGGGAATTGTCGTTATTGGTCGAACAGATCCGCGTTATGTTAGCGAACAGACTATCTAGAACTGGA
>Cuckoo-roller
GTAACTGCTGTATGAGTAGGATTTAAGGATTGAGCGTATTTCTCGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCACCACAGCTTGCCTTCCCATCCGCTGAGGGACAATAATGGGGAAACCGGAATTGTCGTAATCTGTTGAACAGATCCGGCGTATGTTAGCGAACAGACTATCAAGAACTGAA
>Sunbittern
GTGACTGCTGTATGAGTAGAATTTTAGGATTGAGGGTATTCCTTGGGGAGACCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTAACCCCAGGTTACCTTCGCACCCGCTGAGGGACAATAATGAGGAAACCGGAATTGTTAGGACTGGTCGAACAGATCCGGGTTATATTAGTGAGCAGACTATCAAGATCCGGA
>MacQueens-bustard
GTAACTGCTGTATGAATAGAATTTGAGGATTGAGGGTATTTCTTGAGCAGTCCAGCACCGTCAGAAGCTGGAATAATAGGACGCTGCTCAACCCGGGCTACCTTCCCATCCGCTGAGGGACAATCGTGAAGAAACCGGAATTGTCGTAATTGGTCGAAGAGATCCGGCTTAGGTTAACGAACAGACTGTCAAGAACTGGA
>American-crow
ACAACTGCTGGATGAGTAGAATTTGAGGATTGAGGATATTTTTTGAGGAGTCCAGTACCGTCAGAAATTGGAATAATCTAACGCTGATCAATCCAGGTTACCTTCCCACCCGCTGAGGGACTATATTGAGGAAACCGGAATTGTCGTAGTTGGTCGAGCATATTCGGCTTATGTTAGATACCAGACTATCGAGAACTAGA
>Great-crested-grebe
GTAACTGCTGTATGAGTAGAATTTGGGGATTGAGGGTATTTCTTGAGGAGTCCAGCCCCGTCGGAAGCTGGAATAATAGAAAGCTGCTCAACCCGGGTTGCCTTCCCATCCGCTAAGGGACAATAATGAGGAAACCGGAATTGACGTGATTGGGCGAACAGAACTGGCTTATGTTAGCGAACAGACTATCAAGAACTGGA
>Turkey
GTAACCGACGTATGAGTAGAATTTAAAGATTGGGAGTATTTGTTAGGGAGTCGAGCACCGTCGGAAGCTGGATTATTATGACGCTGCTCGACCCGGGCCACCTTCTGGTACGCTCAGGGACTGTAATAACGAGACCAGAATTGCCGTAACTGGTCGACCAGATCCGGCCTATATTCGCGAAAAGGCTACCAACAATTGGA
>Speckled-mousebird
GTAACCACTGTATGAGGAGCATATGAGGATTCAGGGTATCCCTTGAGGAGTCCAACATCGTCAGAAGCTGGAATATTAGGACGCTGCTCAGCCCAGGTTACCTTCCGATCCGCTGAGGGGCAATAATGAGGAGGCCGGAATTGTCGTAATTGGCCGAACAGATGCGACTAATGTTAGCGAACGGACTATCAAGAGCTGCA
>Rifleman
GCCACTGCTGTATAAGTATAATTTGAGGATTGAGGGCGTTTCTTGGGGAGTCCAGCACCGTCCGAAGTTGGAATAATAGAACGCTGCTCAACACAGATTACCTTCCCACCCGCTGAGGGACGATAATGAAGAAACCGGAATTGTCGCAATTGATCGAGCACATCCGGCCTATGTTAGAGACCAAACTATCAAGAAATGAA
>White-tailed-tropicbird
GTAACTGCTGTATGAGTAGAATTTGAGGACTGAGGGTATCTCTTGAGGAGTCCAGAACCGTTAGTAGCTGGAATAATAGAACGCTTCTCAACCCAAGTTGCCTTCCCATCCGCTGAGGGACAATCATGAGGAAAACGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAAACTATCAAGAACTGGA
>Barn-owl
GTAACTGCTGTATGAGTAGAATTTGAGGATTGGGAGTATTTCTTGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCAGGTTACCTTCCCATCTGCTGAGGGACAATAATGAGGAAACCAGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTTGCGAACAAACTATCAAGAACTGGA
>Kea
GTAACTGCTGTATGAATAGAATTTGAAGATTGAGCATATTTCTTAAGGAATCCAGCACCGTCGTAGGGTGGAGTAATAGAACCCTGCTCAACCCAGGTTACCTTCCCACCCGCTGAGGGACAATGAGGAAGAAACCGGAATTGTCGTAGTTGGTTGAACAGCTCCGGCTTATGTTAGGGAACAGACTATCAAGAACTGGA
>Northern-fulmar
GTAACTGCTGTATGAGTAGAATTTGAGGATTGAGGGTATTCCTTGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCAGGTTACCTTCCCATCCGCTGAGGGACAATAATGACGAAACCGGAATTGTCGTGATTGGTCGACTAGATCCGGTTTATGTTAGCGAACAGACTATCAAGAACTGGA
>Hoatzin
GTAACTGCTGTATGAGTAGAATTTGAGGATTGAGGGTATTTCTTGAGGGGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCGGGTTAGCTTCCCATCCGTTGAGGGACAATACTGAGGAAACCGGAATTGTCGTAATTAGTCGAACAGATTCGGTTCATGTTAGCGAACGGACTATTAAGAACTGGA
>Red-throated-loon
GTAACTGCTGGATGAGTAGAATTTGAGGATTAAGGGTATTTCTTAAGGAGTCCAGCACCGTCAGAAGCTGGAATAAGAGAACGCTGCTCAATCCAGGTTACCTTCCCATCCGCTGAGGGACAATAATGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAACTGGA
>Yellow-throated-sandgrouse
GTAACTGCTCTATGAGTGGAATTTGAGGGTTGAGGGCATTTCTTGAGGAGTCCAGCATTGTCAGAAAATGTAATAATAGAACGCTGCCCATCCAGGGTTGTCTTCCCATTCGCTGAGGGACAATAACGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAAATGTA
>White-throated-tinamou
GCAACCGCTTTATGAGGAGCGTTTTAAGGTTGAGGGTATTGTATGAGGAGTCGAACGACATCAGGAGCTGGATCAATAAAACACTGATCAACCCGGGCAACCTTCCCTAACGCTGATGGACTTTCATGGGCGAGCCGCAATTATCGTAATCGGTCCAACAAATCCGGCCTATGGTAACGAACAGGCTATCAACAACTGGG
>Turkey-vulture
GTAACTGCGGCAGGAGTAGAATTTGAGGATTGAGGGTATTTCTTGAGGAGTCCAGCACCGTCAGTAGCTGGAATAATAGAACGCTGCTCAACCCAGGTTACCTTCCGATCCGCTGAGGGACAATAATGAAGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTGTCAAGAACTGGA
>Crested-ibis
GTAACTGCTGTATGAGTAGAATTTGAGGATTGAGGGTATTTCTTGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCAGGTTACCTTCCCATCCGATGAGGGACAATAATGAGGAACCCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATTAAGAACTGGA
>Chuck-wills-widow
ATAACTGCTGTATGAGTAGAATTTGAGGATTGAAGATATTTCTTGAGGAGCCCAGCACCGTCAGAAGCTGGAACAATAGAACGCTGCTCAACCCGCGTCACCTTCCCATCCGTTCAGGGACAATAATGAGGAAACCGGAATTGTCGTAATCGGTCGAACAGATCCGGCATATGTTAGCGGACAGACCATCAAGAACTGGA
>Zebra-finch
ACAACTGTTAGATGAGTAGAATTTGAGGATTGAGGATGTTTCTTGAGGAGTCCAGCACCGTCAGAAGTTGGAATAATCCAACGTTGATCAACTCAGGTTACCTTCCCACCCGCTGGGGGACTATATTGAGGAAACCGGAATTGTCGTAATTGGTTGAGCATATTTGGCTTAAGTTAGATACCAGGCTATCAAGAACTGAA
>Killdeer
GTAACTGCTGTATGAGTAGAATTTGAGGATTGAGGGTATTTCTTGAGGGGTCCAGCACCGTCAGGAGCTGGAATAATAGAACCCTGCTCAACCCGGGTTACCTTCCCGTCCGTTGAGGCACAATACTGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGTTTACATTAGCGAACAGACTATAAAGAACTGTA
>Pigeon
GTAACTGCCGTATCAGTAGAATTTGAGGATTAACGGTATTTTTCTAGGAGTCCAGCACGATCAGAAGCTGGAATAATAGAACGCTGCTCAACTCGGGTTGCCTTCCCTTCCGCTGAGGGACAATAATGAGGAAACCGGCATTGTCGAAATAGGTTGAACAGATCCGGCTTATGTTAGCGAATAGACTATCAAGAACTGGA
>Common-ostrich
GTAACCGCTTTATAAGGAGAGATTAAAAATTAAGGGTATCTCTTAAGGAGTCGAGCACCGTCAGGAGCTGGATCAATAAAAAGTTGATAAACCTGGGTAATCCTCCCAACCCCTGATGGACTCTGATGAGGAAGCCGGTATCATCGTAATTGGTCGAACAGATCCGGCTTATGGTAGCCAACAGGCTATCAACAACTGGG
>White-tailed-eagle
GTAACTGCTGTATGAGTAGAATTTCAGGATTGAGGGTATTTCTTGAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAACCCGGCTCAACCCAGGTTACCTTCCCATCCGCTGAGGGATAATAATGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAATTGGA
>Bar-tailed-trogon
GTAACTGCTGCATGGGTAGGATTTGAGGATTGAGGGTATTCCTTGAGGAGTCCAACACCGTCTTCAGCTGGGATAATAGAACGCTGCTCAACACAGGTTATCTTCCCATCCGCTGAGTGACAATGATGAGGAAACCGATATTGTCGTAACGGGTCGAACAGATGTGGCGTATGTCAGCGAATAGAACATCAAGAACTGAC
>Emperor-penguin
GTAACTGCTGTATGAGTAGAATTTTAGGTTTGAGGGTATTTCTTGAGGAGTCCAGCACCGTCAGTAGCTGGAATAATAGAACGCTGCTCAACCCAGGCTATCTTCCCATCCGCTAAGGGACAATAATGAGGAACCCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTAGGTTAGCGAACAGACTATCAAGAACTGGA
>American-flamingo
GTAATTGCTGTATGAGTAGAATTTAAGGATTGAGGGTATTTCTTGAGGAGTCCAGCCCCGTCAGAAGCTGGAATAATAGAACGTTGCTCAACCCGGGTTGCCTTCCCATCCGCTAAGGGACAATAATGAGGAAACCGGAATTGTCGTGATTGGTCGAACAGATCCGGCTTATGTTACCGAACAGACTATCAAGAACTGGA
>Adelie-penguin
GTAACTGCTGTATGAGTAGAATTTGAGGTTTGAGGGTATTTCTTGAGGAGTCCAGTACCGTCAGAAGCTGGAATAATAGAACGCTGCTCAACCCAGGTTATCTTCCCATCCGCTGAGGGACAATAATGAGGAACCCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAACTGGA
>Chicken
GTAACCGCCGTATTAGTAGAATTTAAAGATTAAGGGTATTTGTTAGGGAGTCGAGCACCGTCGGAAGCTGGAATAGTATGACGCTGCTCGACCCGGGCCACCTTCTGGTACGCTCAGGGACTGTAATAACGAAAGCAGAATTGCCGTAACTGGTCGACCAGATCCGTCCTATATTAGCGAAAAGACTATCAACAATTGGA
>Peregrine-falcon
GTAACTGCTGTATGAGTGGAATTTGAGGATTGAGGGTATTTCTTGGGGAGTCCAGGACCGTCAGAACCTGGAATAATAGAACGCTCCTCAACCCAGGTTACCTTCCCATCTGTTGAGGGACAACAATGAGGAAACCGGAATTGTCGTAATTGGTCGAACAGATCCGGCTTATGTTAGCGAACAGACTATCAAGAACTGGA
>Golden-collared-manakin
GCAACTGCTGTATGAATAGAGTTTGAAAATTAAGGGTGTTTCTTGAGGAGTCCAGCACTCTCAGAAGTTGGAATAAGAGAACGTTGCTCAACCCAGGCTACCTTCCCACCCGCTGAGGGGCTATAATTAGGAAACCGAAATTGTCGTAATTGGTCGAGCATATTCGGCTTATGTTAGAGATCAGACTATCAAGAACTTGA
>Budgerigar
GTAACTGCTGTATGAATAGAATTTGAAGATTGAGGGTATTTCTTAAGGAGTCCAGCACCGTCATAGGCCGGAGTAATAGAACCCTGCTCAACCCAGGTTACCTTCCCACCCGCTGCGGGAAAATGAGGAAGCAACCAGAATTGTCGTAATTGGTTGTACAGCTCCGGCTTATGTTAGAGCACAGACTATCAAGTACTGGA
>Little-egret
GTAACTGCTGTATGAGCAGAATTTGAGCATTGAGGGTATTTCTTGAGGAGTCCAGCACCGTCGGAAGCTGGAGTAAGAGAACGCTGCTCAACCCAGGTTACCTTCCCATCCGCTGATGGACAATAATGAGGAAACCGGAATTGTCGTAACTGGTCGTACAGATCCGGCTTATGTTAGCGAACAGACTGTCAAGAACTGGA
>Red-legged-seriema
GTAACTGCTGTATGAGTAGAATTTGAGGATTGAGGGTATTTCTTAAGGAGTCCAGCACCGTCAGAAGCTGGAATAATAGAATGCTGCTCAACCAAGGTAACCTTCCTATCCGATGAGGGACAATAATGAGGAAACCGGAATTGTAGTTATTGGTCGAACAGATCCGGCTTACGTTAGCGAACAGACTATTAAGAACTGGA
>Brown-mesite
GTAACTGTTGTATGAGTAGAATTTGAGTATTGAGAGTATTTCTTGAGGAGTCCAGCACCGTCAGAAACTGGAATAATACAACGCTGCTCAACTCGGGTTGCCTTCCCATCCGCTGAGGGGCAATAATGAAACAACCAGAATTGCCATAATTGGTCGAACAGACCCGGCTTATGTTAGCGAACGGGCTATCAAGAACTGGA
>Chimney-swift
GTAACTGCCGTGTGAGTAGAATTTGAGAATTGAGGATGCTTCTTGAGTAGTCAGTCATTGTCACAAGCGGGAATAATCGAACGCGGCTCAACTCGGGTTACCTTCCCATCCGCTGAAACACAAGAATGTGGAAACCGGAATTGTTGGAGTTGTCCGAATGAATCCGGCCTACGTTAGCGAACAAACTATCAAGAACTGGA
>Pekin-duck
GTAACCTTCTCATGAGTAAAATATGAGGATTGAGCGTATTTCTTAGGGAGTCCAGTACAATCGGAAGCTGGAGTAGTAGAACTTTGCTCGACCCGGGCTAACTTCGCATCGGATGAGGGACTATAATAAGGAGATCAGAAGTGTCTTAATTGGTCGACCAGATCCGACTGCTGTTAGCGAACAGACTATCAACATCTGGA
>Red-crested-turaco
GTAAGTGCTATATGAGTAGAATTTGAGGATTGAGGGTATTTCTTGAAGAATCCAGCACCGTCAGAAGCTGGGATAATAGAACGCTGCTCAACCCGGGTTACCTTCCCATCCGCTGAGGGACAATAATGAATAAACCGAAATTGTCGTAATGGGTCGAACGGATCCCGCTTATGTTAGCGGACAGACTATCATGAACTGGA
>Rhinoceros-hornbill
GTAACTGCTATATAAGTAGGATTTGAGAATTGAGGGTATTTCTTGAGGAGTCGAGCACCGTCAGAGGCTGAAATAATAGAACGCTGCTCAACCCAGGTTACCTTCCCACCCGCCGATGGACAATAATGAGGAAACCAGCATTGTCGTAATTGGTCAAACAGATCCGGCCTATGTTCGCGAACAGACTATCAAGAACTGGA
>Medium-ground-finch
ACAACTGCTAGATGAGTAGAATTTGAGGATTGAGGATGTTTCTTGAGGAGTCTGGCACCGTCAGAAGTTGGAATAATCCAACGTTGATCAACCCAGGTTACATTCACACCCGCTGGGGGACTATATTGAGGAAACCGGAATTATCGTAATTGGTCGAGCATATTCGGCTTATGTTAGATACCAGGATATCAAGAACTGGA
