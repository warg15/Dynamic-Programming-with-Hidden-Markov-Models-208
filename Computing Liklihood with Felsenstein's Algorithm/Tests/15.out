-3357.776608061112
